src/views/MainPage.vue

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable vue/no-parsing-error */
+<template>
+  <v-container>
+      <div v-for="film in films" :key="film.id" class="card-film">
+      <div class="card-film__img">
+        <img :src="'https://image.tmdb.org/t/p/w500' + film.poster_path" alt="poster">
+      </div>
+      <div class="card-film__descr">
+        <div class="card-film__descr__name">
```

Comment:
strange usage of BEM
element of the element. https://ru.bem.info/methodology/quick-start/#%D0%B2%D0%BB%D0%BE%D0%B6%D0%B5%D0%BD%D0%BD%D0%BE%D1%81%D1%82%D1%8C-1